tasks/findFeatures.js

```diff
@@ -91,16 +97,9 @@ const generateFeatures = ({ locations, report, options }) => {
     foundCount++;
   }
 
-  let foundCount = 0;
-  let featureCollection = {
-    type: 'FeatureCollection',
-    features: []
-  };
-
-  return new Promise(async (resolve, reject) => {
+  return new Promise(async resolve => {
     console.log('⏳ Generating features...');
 
-    const usStates = await fs.readJSON('./coronavirus-data-sources/lib/us-states.json');
```

Comment:
was this intentional?

Reply:
Yep, its not used anywhere. Can easily chuck it back in once its referenced.

Reply:
Ah yes, leftovers.